src/Apps/LlamaAirforce/Pages/Bribes/Util/AuraHelper.ts

```diff
@@ -0,0 +1,42 @@
+import type { OverviewResponse } from "@/Apps/LlamaAirforce/Pages/Bribes/Models";
+
+export const AuraConstants = {
+  HH_API_URL: "https://api.hiddenhand.finance/proposal/aura",
+  LA_API_URL: "https://llama-airforce-api.aura.finance/dollar-per-vlasset",
+  START_ROUND: 28,
+  START_DATE: 1689019200,
+  BIWEEKLY: 60 * 60 * 24 * 14,
+};
+
+// Helper to merge HH data
+export function getMergeWithHiddenHands(
```

Comment:
I think it would be easier if this function took non-promise parameters? Can just move the `Promise.all()` out into `DashboardService.ts`.